config/application.yml

```diff
@@ -0,0 +1,5 @@
+SENDGRID_USERNAME: 'santiramirez'
+SENDGRID_PASSWORD: 'password1234'
+SENDGRID_ADDRESS: 'smtp.sendgrid.net'
+SENDGRID_DOMAIN: 'target.com'
+SENDGRID_PORT: '587'
```

Comment:
this file shouldn't be pushed. You must add this file int he git ignore.
You can add only the vars without their values in the application.yml.example